versions/3.1.1.md

```diff
@@ -99,6 +99,19 @@ Some examples of possible media type definitions:
 The HTTP Status Codes are used to indicate the status of the executed operation. 
 Status codes SHOULD be selected from the available status codes registered in the [IANA Status Code Registry](https://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml).
 
+##### <a name="undefinedAndImplementationDefinedBehavior"></a>Undefined and Implementation-Defined Behavior
+
+This specification deems certain situations to have either _undefined_ or _implementation-defined_ behavior.
+
+Behavior described as _undefined_ is likely, at least in some circumstances, to contradict the specification.
```

Comment:
```suggestion
Behavior described as _undefined_ is likely, at least in some circumstances, to result in outcomes that contradict the specification.
```
Latest change missing

Reply:
I've pushed a fix porting the other commit.